clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/HttpElasticsearchDAO.scala

```diff
@@ -94,24 +93,30 @@ class HttpElasticsearchDAO private[dataaccess] (
     index: ElasticsearchIndex[_]
   ): Future[Boolean] = {
     val indexExistsDefinition = indexExists(index.indexName)
-    httpClient.execute(indexExistsDefinition).map(_.exists)
+    httpClient
+      .execute(indexExistsDefinition)
+      .map(ElasticsearchUtil.unpackResponse(_).exists)
   }
 
   private[dataaccess] def createIndexType(
     index: ElasticsearchIndex[_]
   ): Future[Unit] = {
-    val createIndexDefinition = createIndex(index.indexName) mappings mapping(
-      index.indexType
+    val createIndexDefinition = createIndex(index.indexName).mappings(
+      mapping(index.indexType)
     )
     val replicatedIndexDefinition =
       if (ClioServerConfig.Elasticsearch.replicateIndices)
         createIndexDefinition
       else createIndexDefinition.replicas(0)
-    httpClient.execute(replicatedIndexDefinition) map { response =>
-      if (!response.acknowledged || !response.shards_acknowledged)
-        throw new RuntimeException(s"""|Bad response:
-                                       |$replicatedIndexDefinition
-                                       |$response""".stripMargin)
+    httpClient.execute(replicatedIndexDefinition).map { response =>
+      val unpacked = ElasticsearchUtil.unpackResponse(response)
+      if (!unpacked.acknowledged || !unpacked.shards_acknowledged) {
+        throw new RuntimeException(
```

Comment:
According to codecov, there's no coverage for the negative cases here. If it's easy to add, it would be nice to have.